lesson-6/twenty-one.js

```diff
@@ -34,22 +34,30 @@ function dealCard(deck) {
   } else if (randomSuit === 'spades') {
     card.unshift('♠️');
   }
-  return card;
+  hand.push(card);
 }
 
 function initialDeal(deck, playerHand, dealerHand) {
-  playerHand.push(dealCard(deck));
-  dealerHand.push(dealCard(deck));
-  playerHand.push(dealCard(deck));
-  dealerHand.push(dealCard(deck));
+  dealCard(deck, playerHand);
+  dealCard(deck, dealerHand);
+  dealCard(deck, playerHand);
+  dealCard(deck, dealerHand);
 }
 
 function hitOrStay() {
   prompt('Would you like to hit or stay?');
-  return readline.question().toLowerCase();
+  let answer = readline.question().toLowerCase();
+  while (answer !== 'hit' &&
+         answer !== 'h' &&
+         answer !== 'stay' &&
+         answer !== 's') {
+    prompt('Invalid input. Would you like to "hit" or "stay"?');
+    answer = readline.question();
+  }
+  return answer;
 }
 
-function displayHand(hand) {
+function joinAnd(hand) {
```

Comment:
changed function name to joinAnd to better reflect what it does, and because new function was created to display hand